src/core/utils/helpers/fetcherHelper.ts

```diff
@@ -0,0 +1,19 @@
+import { store } from "../../store";
+
+interface ConfigPossibilities {
+  fbs: string;
+  publizon: string;
+}
+
+export const configTypes: ConfigPossibilities = {
+  fbs: "fbsBaseUrl",
+  publizon: "publizonBaseUrl"
+};
```

Comment:
```suggestion
export const configTypes: ConfigPossibilities = {
  fbs: "fbsBaseUrl",
  publizon: "publizonBaseUrl"
} as const;
```

Reply:
If you write `as const` you make sure that the object is immutable and emphasize that it is meant for static "configuration" usage.